app/controllers/concerns/idv/ab_test_analytics_concern.rb

```diff
@@ -18,6 +18,11 @@ def ab_test_analytics_buckets
         buckets = buckets.merge(lniv_args)
       end
 
+      buckets.merge(
+        ab_testing_idv_ten_digit_otp_bucket:
+                               AbTests::IDV_TEN_DIGIT_OTP.bucket(current_user.uuid),
+      )
+
       buckets.merge(acuant_sdk_ab_test_analytics_args)
```

Comment:
this result is going to be ignored, either:
1. we need to run `merge!` to mutate `buckets`
2. we need to re-assign `buckets
3. (my preferred option) we combine multiple merges into one step

```suggestion
      buckets.merge(
        acuant_sdk_ab_test_analytics_args,
        ab_testing_idv_ten_digit_otp_bucket:
                               AbTests::IDV_TEN_DIGIT_OTP.bucket(current_user.uuid),
      )
```

Reply:
I re-worked this, and yes it's still pretty ugly. LG-12451 calls to clean up all the A/B test logging, making it all look like:
```
ab_tests: {
  test_name: {
    bucket: ...,
    other_params: ...,
  },
  ...,
}
```